src/ui/components/ExplorerHome/ExplorerSimulator.js

```diff
@@ -0,0 +1,773 @@
+// @flow
```

Comment:
I don't see where this simulation view is utilized yet. Can we defer pushing it until it's actually reviewable?

Reply:
oops, that's supposed to be deleted!